source/switching_to_live/before_you_switch_to_live/index.html.md.erb

```diff
@@ -9,22 +9,28 @@ Before you start taking live payments, you should:
 
 * have a signed MOU or contract with GOV.UK Pay
 
-* [provide organisation information](#provide-organisation-information) to GOV.UK Pay
+* [provide organisation information](#provide-organisation-name-and-address) to GOV.UK Pay
 
 * request a [live GOV.UK Pay account](#request-a-live-gov-uk-pay-account)
 
 * make sure that the right people on your
 team [know how to report an emergency](#emergency-contact-details)
 
-GOV.UK Pay currently supports live gateways for the PSPs
+GOV.UK Pay currently supports live gateways for the payment service providers (PSPs)
 [ePDQ](/switching_to_live/set_up_a_live_epdq_account),
 [Smartpay](/switching_to_live/set_up_a_live_smartpay_account) and
 [Worldpay](/switching_to_live/set_up_a_live_worldpay_account).
 
-## Provide organisation information
+GOV.UK Pay also supports a live gateway for Stripe.
+
+How you switch to live differs depending on whether you use Stripe or another PSP.
```

Comment:
Could it be worth explaining why? E.g. because gov.uk pay sets up the merchant account with Stripe once the following details are entered, whereas account setup with the other PSPs happens offline

Reply:
Maybe, but I don't have strong feelings one way or another. If you know that the user would really appreciate this information then let's include it, otherwise I would err on the side of less rather than more.

Reply:
sure, let's stick with less then

Reply:
ok will leave as is